source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -149,21 +149,18 @@ const computeCaseCardComponent = (
   const selfHasSigned = casePersonData?.hasSigned;
   const isCoApplicant = casePersonData?.role === "coApplicant";
 
-  const isEncrypted = answersAreEncrypted(currentForm.answers);
-  const shouldEnterPin = isEncrypted && isCoApplicant && isWaitingForSign;
-
-  const shouldShowCTAButton =
-    (!isEncrypted || shouldEnterPin) &&
-    (isCoApplicant
-      ? isWaitingForSign && !selfHasSigned
-      : isOngoing ||
-        isNotStarted ||
-        isRandomCheckRequired ||
-        isSigned ||
-        isClosed ||
-        isVivaCompletionRequired ||
-        isActiveSubmittedRandomCheck ||
-        activeSubmittedCompletion);
+  const shouldEnterPin = answersAreEncrypted(currentForm.answers);
+
+  const shouldShowCTAButton = isCoApplicant
+    ? isWaitingForSign && !selfHasSigned
+    : isOngoing ||
+      isNotStarted ||
+      isRandomCheckRequired ||
+      isSigned ||
```

Comment:
Created a new card since it involves a few extra scattered files to properly remove and I think that will bloat this PR.

https://app.clickup.com/t/3gpdxmt